agent/command.go

```diff
@@ -250,21 +250,19 @@ func (a *Agent) runCommand(ctx context.Context, tc *taskContext, logger client.L
 		tc.taskConfig.Expansions.Put(key, newVal)
 	}
 	defer func() {
-		if !tc.unsetFunctionVarsDisabled || tc.taskConfig.Project.UnsetFunctionVars {
```

Comment:
Since the project YAML `unset_function_vars` now is a no-op, is there a ticket to delete it?

Reply:
This PR deletes that too.

Reply:
Thanks for clarifying! In that case, we should also delete it from the [self-tests YAML](https://github.com/evergreen-ci/evergreen/blob/1a97b4a87851cab70bd3f28ad131da9da7fb427a/self-tests.yml#L1).

Reply:
done